src/isar/storage/test_download_blob.py

```diff
@@ -0,0 +1,42 @@
+import os
+from isar.config.keyvault.keyvault_service import Keyvault
+from isar.config.settings import settings
+from isar.storage.blob_storage import BlobStorage
+
+
+class TestBlobdownload:
```

Comment:
Not sure if we want to have this test in the repo. Do we want it to run in CI/CD?